server/controllers/follow.controllers.js

```diff
@@ -44,20 +48,20 @@ const followController = {
           followee_id: followeeId,
           follower_id: followerId,
         });
-
+        const userDetails = profileHelper.profiler(findUnFollowee);
         return res.status(201).json({
-          status: 201,
-          message: `Successfully followed user`,
+          message: 'You have Successfully followed user',
```

Comment:
`Successfully` should start with a lower case letter

Reply:
And the message can be - `You have successfully followed this user`

Reply:
Noted and fixed